training/utils/llama_patch.py

```diff
@@ -38,16 +38,20 @@ def forward(
 
     attention_mask: [bsz, q_len]
     """
+    if self.config.pretraining_tp != 1:
+        raise ValueError("Only model.config.pretraining_tp=1 is supported but is", self.config.pretraining_tp)
```

Comment:
wouldn't incude it here, slows down things and all pretrained checkpoints have the value as `1` on the hub.

Reply:
sure, just removed it